packages/web-workers/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@remote-ui/web-workers",
-  "version": "1.2.9",
+  "version": "1.2.10",
```

Comment:
There is technically a new feature in here so IMO this should be `1.3.0` instead

Reply:
Fixed!